src/poliastro/twobody/orbit.py

```diff
@@ -162,9 +162,12 @@ def from_body_ephem(cls, body, epoch=None):
             warn("Input time was converted to scale='tdb' with value "
                  "{}. Use Time(..., scale='tdb') instead."
                  .format(epoch.tdb.value), TimeScaleWarning)
+        if get_velocity:
```

Comment:
I still would remove this functionality, and make the user responsible of setting the correct ephemerides from the outside.